File: CCL-example-java/src/main/java/hohserg/ccl/example/ClientProxy.java
package hohserg.ccl.example;

import codechicken.lib.packet.PacketCustom;
import net.minecraftforge.fml.common.FMLCommonHandler;
import net.minecraftforge.fml.common.event.FMLPreInitializationEvent;

public class ClientProxy extends CommonProxy {
    public void preinit(FMLPreInitializationEvent e) {
        super.preinit(e);
        PacketCustom.assignHandler(Main.modid, new ClientPacketHandler());
        ClientUpdate.init();
    }
}

File: CCL-example-scala/src/main/scala/hohserg/ccl/example/ClientProxy.scala
package hohserg.ccl.example

import codechicken.lib.packet.PacketCustom
import net.minecraftforge.fml.common.event.FMLPreInitializationEvent

class ClientProxy extends CommonProxy{
  override def preinit(e: FMLPreInitializationEvent): Unit = {
    super.preinit(e)
    PacketCustom.assignHandler(Main.modid, new ClientPacketHandler)
    ClientUpdate.init()
  }
}
